### DISCUSSION TITLE
[2022/03/08] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#195] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/194 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by chloe463** [comment]
## 続・Tailwindcss で遊んでる

- 慣れてくると結構サクサク書ける
- Wantedly design system の elvation を再現する css class を tailwindcss 上でつくるための plugin 書けた
- https://github.com/chloe463/playground/blob/change/stack/tailwind/plugin/elevation.js
- styled-components/macro 使ってると、css class 名がその styled-components の名前になって chrome devtools からどこのコードか見つけやすいが、tailwindcss だとその手がかりがなくなって見つけづらい問題があるかも
	- 自前でそれぞれに classname つけるとか、id つけるとか解決策がないわけではないがちょっと面倒になるかも

**Comment by kobayang** [comment]
![image](https://user-images.githubusercontent.com/12181179/157212561-1c7d91ff-d1f4-4eff-bee0-f03d62299453.png)

https://kentcdodds.com/blog/state-colocation-will-make-your-react-app-faster
